recipes-samples/images/rpb-minimal-image.bb

```diff
@@ -1,6 +1,6 @@
 SUMMARY = "Minimal image"
 
-IMAGE_FEATURES += "splash tools-debug"
+IMAGE_FEATURES += "splash tools-debug debug-tweaks"
```

Comment:
this change is not explained in the commit. and it's a change in -minimal image. I am not saying it's a bad change. but it shouldn't be done in this commit. 

ideally this commit should be a NO-OP , and no image should be modified after this factorization.

Reply:
e.g. you need to build all images before and after your commit and check buildhistory to make sure nothing changed.

Reply:
Yes, i will add this change into a separate commit after this one, regarding the testing i didn't do a full build of all images only minimal and console for the other ones i reviewed the variables involved by this change using bitbake -e.